WordPress/src/main/java/org/wordpress/android/ui/TextInputDialogFragment.java

```diff
@@ -78,9 +79,14 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
             hintView.setVisibility(View.GONE);
         }
 
-        if (!isMultiline) {
-            editText.setMaxLines(1);
+        if (isMultiline) {
+            editText.setSingleLine(false);
+            editText.setMaxLines(10);
+            editText.setHorizontallyScrolling(false);
+        } else {
+            editText.setSingleLine(true);
         }
+
```

Comment:
There's a pre-existing bug in the code below this that causes the text to be scrolled to the end, which is even more noticeable with multiple lines. Any chance you'd want to look into fixing that here?

Reply:
When the text is taller than `maxLines`, the `EditText` will become scrollable scrollable. Is that OK?

https://github.com/user-attachments/assets/bf0dd546-ad6e-4cc7-9778-29bd09ef5896

Reply:
@Agoni-0 Having it scrollable is totally fine, it's just that when you enter that screen, it's scrolled to the bottom instead of the top. I believe this is due to the text being selected?

Reply:
When I open the dialog, it's scrolled to the bottom automatically. Do you mean to keep the text at the top when opening？

https://github.com/user-attachments/assets/f378e20c-fbd6-4183-aa6f-c90109328a07

Besides, the `EditText` style seems not suitable for multi line text. Can I add a new style like thie？
![image](https://github.com/user-attachments/assets/8c47300f-d9b8-444b-92ce-e94f6c7a0bd9)

Reply:
> Do you mean to keep the text at the top when opening？

Yes.

Reply:
> Besides, the EditText style seems not suitable for multi line text. Can I add a new style like thie？

Can you elaborate on what the style change is?

Reply:
> > Do you mean to keep the text at the top when opening？
> 
> Yes.

Thank you for your prompt, I have completed this modification.


https://github.com/user-attachments/assets/b3ec911e-d31c-4c9e-9ee0-529ebf9c68a9

Reply:
> > Besides, the EditText style seems not suitable for multi line text. Can I add a new style like thie？
> 
> Can you elaborate on what the style change is?

I want to remove the bottom horizontal line when typing multiple lines and use a text area that is different from the background color of the message box as input. Here is a small demo.
![image](https://github.com/user-attachments/assets/ad4d087b-a257-472f-a17f-e8c1e2acabc7)
![image](https://github.com/user-attachments/assets/b4316d9d-bb22-4005-bdd2-4fe1c0c24a48)

Reply:
@Agoni-0 I think that's a nice change!

Reply:
Thank you for your recognition. I have finished.

Reply:
@Agoni-0 Unfortunately, there are a couple of problems with the background. First, the drawable file is missing the `.xml` extension, which prevents the app from building. Second, the background color doesn't work for dark mode (see screenshot below). Let's just drop the background drawable and stick with the default style.

![darkmode](https://github.com/user-attachments/assets/a6cf7b28-18ed-499a-95bf-ba516c211011)

Reply:
I'm sorry, I will pay attention to these problems in my future work and avoid them from happening again.

Reply:
I have dropped the background drawable